addon/serializers/children.js

```diff
@@ -0,0 +1,12 @@
+import OsfSerializer from './osf-serializer';
+
+export default OsfSerializer.extend({
+    serialize: function(snapshot, options) {
```

Comment:
Okay, after talking with @abought, he's right, I just need to upload the payloadKeyFromModelName hook on the serializer, which sets the type.